doc/gui/examples/controls/column_name_styling.py

```diff
@@ -0,0 +1,26 @@
+# Example for column name styling
+
+from taipy.gui import Gui
+import pandas as pd
+
+# Sample data in DataFrame format
+df = pd.DataFrame({
+    "Name": ["Alice", "Bob", "Charlie"],
+    "Age": [25, 30, 35],
+    "Occupation": ["Engineer", "Doctor", "Artist"]
+})
+
+# Define column properties
+columns = [
+    {"field": "Name", "header": "Name", "header_style": "color: blue; font-weight: bold;"},
+    {"field": "Age", "header": "Age", "header_style": "color: blue; font-weight: bold;"},
+    {"field": "Occupation", "header": "Occupation", "header_style": "color: blue; font-weight: bold;"}
+]
+
+# Page content with table and header styling
+page = """
+<|table|data={df}|columns={columns}|>
+"""
```

Comment:
with columns names showing spaces or transformed characters:

```py
df = pd.DataFrame({
    "Name": ["Alice", "Bob", "Charlie"],
    "Age": [25, 30, 35],
    "Occupation or Job": ["Engineer", "Doctor", "Artist"]
})
```

you can define style at the page level like this :
```py
page = Markdown("""4
<|table|data={df}|columns={columns}|>
""", style={".taipy-table-name": {"color": "blue"}, ".taipy-table-occupation-or-job": {"color": "green"}})
```

Reply:
I think `page = Markdown("""4` the 4 here is not necessary but i have still added it. If it was by mistake i will remove it 😁